runtime/gc_realtime/RealtimeAccessBarrier.hpp

```diff
@@ -156,6 +156,12 @@ class MM_RealtimeAccessBarrier : public MM_ObjectAccessBarrier
 	virtual I_32 backwardReferenceArrayCopyIndex(J9VMThread *vmThread, J9IndexableObject *srcObject, J9IndexableObject *destObject, I_32 srcIndex, I_32 destIndex, I_32 lengthInSlots);
 	virtual I_32 forwardReferenceArrayCopyIndex(J9VMThread *vmThread, J9IndexableObject *srcObject, J9IndexableObject *destObject, I_32 srcIndex, I_32 destIndex, I_32 lengthInSlots);
 
+	virtual IDATA
+	indexableDataDisplacement(J9VMThread *vmThread, J9IndexableObject *src, J9IndexableObject *dst)
+	{
+		Assert_MM_unreachable();
```

Comment:
compile error:

```
RealtimeAccessBarrier.hpp:163:2: error: control may reach end of non-void function [-Werror,-Wreturn-type]
```

I guess, we cannot inline this assert call?

Reply:
I think we need to default return (such as return 0;)

Reply:
right, that's more correct interpretation